svf/lib/Util/extapi.c

```diff
@@ -0,0 +1,734 @@
+#define NULL ((void *)0)
+
+/*
+   The functions in extapi.bc are named using two patterns:
+    1: "svf_methodName";
+    2: "svf_methodName_methodProperties";
```

Comment:
Make sure use the below format methodname_propety_orginalname

Reply:
Some function names are substrings of other function names, such as "malloc" and "safe_malloc," there is an issue when matching app function names with extapi function names. This problem results in one app function name matching multiple extapi function names. To avoid this problem, placing the prefix "svf_" in front of the function names, such as "svf_malloc" and "svf_safe_malloc," can prevent this issue.